base/logging_context.go

```diff
@@ -55,6 +55,8 @@ type LogContext struct {
 	//RequestPort string
 	//// RemoteAddr is the IP and port of the remote client making the request associated with this log
 	//RemoteAddr string
+
+	implicitDefaultCollection bool
```

Comment:
Adding a comment for this property would be nice - it's unclear from name alone what it indicates.